client/src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, {Component} from 'react';
 import './App.css';
 import TopCharts from './Clusters/TopCharts/TopCharts.js';
-import MusicApps from './Clusters/MusicApps/MusicApps.js';
+import StandardCluster from './Clusters/StandardCluster/StandardCluster.js';
```

Comment:
file name should be snack case. can you update the file names for the files that you touched in this PR?